server/constants/vars.js

```diff
@@ -45,6 +45,7 @@ export const DEFAULT_CHART_FILTERS = {
     { name: 'Female', value: TRUE_STRING },
     { name: 'Missing', value: TRUE_STRING },
   ],
+  sites: [],
```

Comment:
should we update `DEFAULT_CHART_FILTERS` so that it includes all the sites by default?

Reply:
A user is only allowed to view data from the sites they belong to, if we include all of the sites here then users without access to that site will view that data.

Reply:
This constant is initiated before we become aware of the sites a user has access too, it would only be able to be added after we've processed the initial graph, but at that time we are using the query params that are sent by the ui.

Reply:
right but we could make it a function that gets a user passed to it and returns the data we need

Reply:
I've opted for another path [218cfc9](https://github.com/AMP-SCZ/dpdash/pull/441/commits/218cfc99b61b8fbafa89ab15275c64beca1f0b10)